### PR TITLE
Update readme & changlog

### PR DESCRIPTION
**Pull request by heavengate**
**Update readme & changlog**
**TODO:** refine picture and add mobile map_fps in next pr

### CODE REVIEW

**Review by qingqing01 — DISMISSED** [review]

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,37 @@
...
+  - 检测：
+    - 新增Swin-Transformer目标检测模型
+    - 新增TOOD(Task-aligned One-stage Object Detection)模型
+    - 新增GFL(Generalized Focal Loss)目标检测模型
+    - 发布Sniper小目标检测优化方法，支持Faster RCNN及PP-YOLO系列模型
+    - 发布针对EdgeBoard优化的PP-YOLO-EB模型
+
+  - 跟踪
+    - 发布fairmot 高精度模型、小尺度模型和轻量级模型
+    - 发布行人、人头和车辆实跟踪模型库，覆盖航拍监控、自动驾驶、密集人群、极小目标等场景
```
Comment (jerrywgz) [inline docs/CHANGELOG.md]: 发布行人、人头和车辆跟踪垂类模型库

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,37 @@
...
+    - DeepSORT模型适配PP-YOLO, PP-PicoDet等更多检测器
+
+  - 关键点
+    - 新增Lite HRNet模型
+    - 关键点检测适配Paddle Lite部署
```
Comment (jerrywgz) [inline docs/CHANGELOG.md]: 这里和预测部署重复了，删掉吧

File: docs/CHANGELOG.md (hunk 1 above)
Comment (heavengate) [reply docs/CHANGELOG.md]: Done, thanks!

File: docs/CHANGELOG.md (hunk 2 above)
Comment (heavengate) [reply docs/CHANGELOG.md]: Done, thanks!

**Review by jerrywgz — APPROVED** [review]

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,36 @@
...
+- 模型丰富度:
+  - 检测：
+    - 新增Swin-Transformer目标检测模型
+    - 新增TOOD(Task-aligned One-stage Object Detection)模型
+    - 新增GFL(Generalized Focal Loss)目标检测模型
+    - 发布Sniper小目标检测优化方法，支持Faster RCNN及PP-YOLO系列模型
+    - 发布针对EdgeBoard优化的PP-YOLO-EB模型
+
+  - 跟踪
+    - 发布fairmot 高精度模型、小尺度模型和轻量级模型
```
Comment (qingqing01) [inline docs/CHANGELOG.md:20]: fairmot  -> FairMOT

**Review by qingqing01 — APPROVED** [review]

File: docs/CHANGELOG.md (hunk 3 above)
Comment (heavengate) [reply docs/CHANGELOG.md:20]: fix this in next PR

**Pull request merged**